WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java

```diff
@@ -288,7 +288,7 @@ public void onClick(View view) {
             holder.totalsTextView.setText(getSinceLabel(currentRowData.getDateSubscribed()));
 
             // Avatar
-            holder.networkImageView.setImageUrl(PhotonUtils.fixAvatar(currentRowData.getAvatar(), mResourceVars.headerAvatarSizePx), WPNetworkImageView.ImageType.AVATAR);
+            holder.networkImageView.setImageUrl(GravatarUtils.fixGravatarUrl(currentRowData.getAvatar(), mResourceVars.headerAvatarSizePx), WPNetworkImageView.ImageType.AVATAR);
```

Comment:
I think we should use "404" for two reasons:
- The default icon is not downloaded from the Gravatar server, instead the local icon is used in WPNetworkImageView.
- (most important) if I remember correctly the Gravatar default Mistery Man is square. WPNetworkImageView.ImageType.AVATAR is circular. We want circular pictures here.